src/components/ViewEnvironments/EnvironmentList/index.tsx

```diff
@@ -1,19 +1,37 @@
-import { Container, InputAdornment, TextField, Tooltip } from "@mui/material";
-import HelpOutlineIcon from '@mui/icons-material/HelpOutline';
+import { Box, Checkbox, Container, FormControlLabel, FormGroup, InputAdornment, Link, TextField, Typography } from "@mui/material";
 import { useApolloClient, useQuery } from "@apollo/client";
 import EnvironmentTable from "../EnvironmentTable";
 import { ALL_ENVIRONMENTS } from "../../../queries";
-import { useEffect, useState } from "react";
+import { useContext, useEffect, useState } from "react";
+import { HelpIcon } from "../../HelpIcon";
+import { UserContext } from "../../UserContext";
+import { useLocalStorage } from "@uidotdev/usehooks";
 
 const SECOND = 1000;
 const MAX_REFETCH_INTERVAL = 10 * SECOND;
 
+function compareStrings(a: string, b: string) {
```

Comment:
String.localeCompare should be able to replace this function (I think it has a case sensitivity setting, though I may be misremembering).

Reply:
aha! I wondered if there was something convenient like that